src/shapes/IText/ITextKeyBehavior.ts

```diff
@@ -64,43 +64,41 @@ export abstract class ITextKeyBehavior<
     const doc =
       (this.canvas && getDocumentFromElement(this.canvas.getElement())) ||
       getFabricDocument();
-    this.hiddenTextarea = doc.createElement('textarea');
-    this.hiddenTextarea.setAttribute('autocapitalize', 'off');
-    this.hiddenTextarea.setAttribute('autocorrect', 'off');
-    this.hiddenTextarea.setAttribute('autocomplete', 'off');
-    this.hiddenTextarea.setAttribute('spellcheck', 'false');
-    this.hiddenTextarea.setAttribute('data-fabric', 'textarea');
-    this.hiddenTextarea.setAttribute('wrap', 'off');
-    const style = this._calcTextareaPosition();
+    const attributes = {
```

Comment:
don't you prefer to inline the arg if it is used only once?

Reply:
sometimes i have some readibility concenrn but here seems fine to do so.